src/main/java/uk/gov/pay/api/service/ConnectorUriGenerator.java

```diff
@@ -97,4 +97,9 @@ public String eventsURI(Account account, Optional<ZonedDateTime> toDate, Optiona
 
         return buildConnectorUri(account, "/v1/events", params);
     }
+
+    public String captureURI(Account account, String chargeId) {
```

Comment:
It might be useful to have a unit test for this method, like the others in this class.